docs/source/compilation.rst

```diff
@@ -36,12 +37,21 @@ In order to build Cartographer ROS, we recommend using `wstool <http://wiki.ros.
 <http://wiki.ros.org/rosdep>`_. For faster builds, we also recommend using
 `Ninja <https://ninja-build.org>`_.
 
+On Ubuntu Focal with ROS Noetic to install these tools:
```

Comment:
...*use these commands* to install these tools...?

Reply:
PTAL